.pre-commit-config.yaml

```diff
@@ -1,33 +1,36 @@
 repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.4.0
+    hooks:
+      - id: check-builtin-literals
+      - id: check-added-large-files
+      - id: check-case-conflict
+      - id: check-toml
+      - id: check-yaml
+      - id: debug-statements
+      - id: end-of-file-fixer
+      - id: forbid-new-submodules
+      - id: trailing-whitespace
 
   - repo: https://github.com/psf/black
-    rev: stable
+    rev: 20.8b1
```

Comment:
Never point pre-commit at a moving tag/branch! It was designed to into its caching system. Instead, run pre-commit autoupdate or turn on pre-commit.ci, which updates weekly as needed via PR.